code/stable/projectile/projectile_u_p_l_b_b_c_d/src/cpp/Projectile.cpp

```diff
@@ -155,68 +155,6 @@ int main(int argc, const char *argv[]) {
     return 0;
 }
 
-double func_t_flight(InputParameters &inParams) {
```

Comment:
while I'm completely fine with the logging of the inline version being much less verbose, we shouldn't lose quite so much logging. Issue six?

Reply:
Issue six indeed!